### PR TITLE
Added external field loading from file to picmi.py

### PR DESCRIPTION
**Pull request by camille12225**

### CODE REVIEW

File: Python/pywarpx/picmi.py [hunk 1]
```diff
@@ -1116,6 +1116,19 @@ def initialize_inputs(self, laser):
...
             ) / constants.c
 
+class LoadInitialField(picmistandard.base._ClassWithInit):
+    def __init__(self, read_fields_from_path,load_B=True, load_E=True, **kw) :
+        self.load_B=load_B
+        self.load_E=load_E
+        self.read_fields_from_path = read_fields_from_path
+
+    def initialize_inputs(self):
+        print(self.read_fields_from_path)
```
Comment (roelof-groenewald) [inline Python/pywarpx/picmi.py]:
```suggestion
```

**Review by roelof-groenewald — APPROVED** [review]
Thanks a lot! Looks good to me.

File: Python/pywarpx/picmi.py [hunk 2]
```diff
@@ -1116,6 +1116,18 @@ def initialize_inputs(self, laser):
                 (self.position[2] - laser.centroid_position[2])**2
             ) / constants.c
 
+class LoadInitialField(picmistandard.base._ClassWithInit):
+    def __init__(self, read_fields_from_path,load_B=True, load_E=True, **kw) :
+        self.load_B=load_B
```
Comment (dpgrote) [inline Python/pywarpx/picmi.py]: This is a detail, but elsewhere, there are spaces around the "=".

File: Python/pywarpx/picmi.py [hunk 3]
```diff
@@ -1116,6 +1116,18 @@ def initialize_inputs(self, laser):
                 (self.position[2] - laser.centroid_position[2])**2
             ) / constants.c
 
+class LoadInitialField(picmistandard.base._ClassWithInit):
+    def __init__(self, read_fields_from_path,load_B=True, load_E=True, **kw) :
```
Comment (dpgrote) [inline Python/pywarpx/picmi.py]: Can you add documentation here? Follow the format use in other classes.

**Comment by RemiLehe** [comment]
@dpgrote @camille12225 @roelof-groenewald  Would it make sense to first add this feature in the [PICMI repository](https://github.com/picmi-standard/picmi), and then use the PICMI class in WarpX, as is done for the other WarpX Python classes.

**Comment by dpgrote** [comment]
@RemiLehe I think adding this to PICMI first is a good idea, more than anything so it doesn't get forgotten. The best place for this new class there would be in the `fields.py` file.

**Comment by roelof-groenewald** [comment]
I see `picmistandard.PICMI_LoadInitialField` was added in https://github.com/picmi-standard/picmi/pull/101.
@camille12225 I believe the failing CI tests will pass if you merge or rebase with `development`, since Remi recently updated the picmi version in #3959.

**Comment by roelof-groenewald** [comment]
Oh, sorry, I see now that the needed changes in picmistandard were not included in the last version update. So this will require another new release of picmistandard. @dpgrote would you like us (@camille12225) to perform that new release (can we even do it?), or do you need to do it?
@camille12225 once the new picmistandard version is released, the WarpX version requirement for picmistandard will need to be updated again (similar to the PR from Remi that I linked above).

**Comment by dpgrote** [comment]
The picmistandard version has been updated to 0.25.0, including PICMI_LoadGriddedField.

**Review by dpgrote — APPROVED** [review]
This looks good! Thanks!

**Comment by dpgrote** [comment]
I've approved this, but something extra that would be nice is a CI test. Perhaps a PICMI version of Examples/Tests/LoadExternalField/inputs_3d could be added.

File: Examples/Tests/LoadExternalField/PICMI_inputs_3d.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+nz = max_grid_size
+
+xmin = -1
+xmax = 1
+ymin = xmin
+ymax = xmax
+zmin = 0
+zmax = 5
+
+plasma_density = 1e21
```
Comment (dpgrote) [inline Examples/Tests/LoadExternalField/PICMI_inputs_3d.py]: These don't seem to be used in the input file.

**Review by dpgrote — APPROVED** [review]
Looks good!

**Pull request merged**